reactors/fcs-tasbe-tacc/src/process_experiment.m

```diff
@@ -30,40 +30,44 @@
 
   
 
-  AGP =AutogateParameters()
+  AGP =AutogateParameters();
 
+  'calling autogate'
+  color_model.blank_file
   autogate = GMMGating(color_model.blank_file,AGP,'plots');
 
-  color_model
+  'done autogate'
+  color_model;
 
-  CM = ColorModel(color_model.bead_file, color_model.blank_file, channels, color_files, color_pair_files)
+  CM = ColorModel(color_model.bead_file, color_model.blank_file, channels, color_files, color_pair_files);
   %CM = set_bead_plot(CM,0)
-  CM = set_translation_plot(CM,color_model.translation_plot)
-  CM = set_noise_plot(CM,color_model.noise_plot)
-  CM = set_bead_model(CM,color_model.bead_model)
+  CM = set_translation_plot(CM,color_model.translation_plot);
+  CM = set_noise_plot(CM,color_model.noise_plot);
+  CM = set_bead_model(CM,color_model.bead_model);
   CM=set_translation_channel_min(CM,color_model.channel_mins);
 
-  CM = set_bead_batch(CM, color_model.bead_batch)
-  CM = set_FITC_channel_name(CM,color_model.fitc_channel_name)
+  CM = set_bead_batch(CM, color_model.bead_batch);
+  CM = set_FITC_channel_name(CM,color_model.fitc_channel_name);
   
   settings = TASBESettings();
 
   output.plots_folder
-  settings = setSetting(settings,'path',output.plots_folder)  
+  settings = setSetting(settings,'path',output.plots_folder);  
   CM = add_filter(CM,autogate);
+  'resolving'
   CM=resolve(CM, settings);
-
+  'done resolve'
   
   experimentName = output.title; 
-  bins = BinSequence(4,0.1,10,'log_bins');
+  bins = BinSequence(0,0.1,10,'log_bins');
   
   
   % Designate which channels have which roles
   AP = AnalysisParameters(bins,{});
   % Ignore any bins with less than valid count as noise
   AP=setMinValidCount(AP,100');
   % Ignore any raw fluorescence values less than this threshold as too contaminated by instrument noise
-  AP=setPemDropThreshold(AP,5');
+  AP=setPemDropThreshold(AP,0');
```

Comment:
You might want to undo this change --- that's what's bringing all the negative data into the bottom bin, and it can lead to distortion in analysis.

Reply:
I'll roll these parameters into the json file instead of hardcoding these values.

Reply:
Good idea.  They're all configurable parameters.  At some point, they will get systematized to make it easier to track what they all are and what their defaults are.